unattended_installer/install_functions/dashboard.sh

```diff
@@ -26,7 +26,11 @@ function dashboard_configure() {
             ip=${dashboard_node_ips[pos]}
         fi
 
-        echo 'server.host: "'${ip}'"' >> /etc/wazuh-dashboard/opensearch_dashboards.yml
+        if [[ "${ip}" != "127.0.0.1" ]]; then
+            echo 'server.host: "'${ip}'"' >> /etc/wazuh-dashboard/opensearch_dashboards.yml
```

Comment:
Change the `echo` to `sed`.

Reply:
The file ``dashboard_unattended_distributed.yml`` does not include the variable ``server.host``. That is why it is better to use ``echo >>`` instead of ``sed`` in this case.